hassio_aw2m/config.yaml

```diff
@@ -0,0 +1,32 @@
+name: AmbientWeather2MQTT
+version: "1.0.0"
+slug: hassio_aw2m
+description: This package listens for local data from Ambient Weather stations (such as the WS-2902C) and converts the incoming data to MQTT events.
+startup: application
+boot: auto
+ports:
+  8132/tcp: 7000
+options:
+  MQTT_SERVER: ""
+  MQTT_USERNAME: ""
+  MQTT_PASSWORD: ""
+  MQTT_REJECT_UNAUTHORIZED: true
+  STATION_MAC_ADDRESS: ""
+  TZ: "America/Chicago"
+  VERBOSE: false
+schema:
+  MQTT_SERVER: "str"
+  MQTT_USERNAME: "str"
+  MQTT_PASSWORD: "str"
```

Comment:
To piggyback on the topic of MQTT config. For other ____2mqtt addons, no config is necessary if you are using the HA Mosquito addon as well. I am not sure how that is accomplished (I can research), but it might be something to look into